lib/gems/pending/appliance_console/database_replication_primary.rb

```diff
@@ -9,7 +9,7 @@ class DatabaseReplicationPrimary < DatabaseReplication
 
     REGISTER_CMD = 'repmgr master register'.freeze
 
-    def initialize
+    def initialize(_hash = {})
```

Comment:
Same thing here. You need to either do something with the hash contents or set the attributes using the accessors on the instance.